python/ray/_private/memory_monitor.py

```diff
@@ -38,9 +38,12 @@ def get_message(used_gb, total_gb, threshold):
         pids = psutil.pids()
         proc_stats = []
         for pid in pids:
-            proc = psutil.Process(pid)
-            proc_stats.append((get_rss(proc.memory_info()), pid,
-                               proc.cmdline()))
+            try:
+                proc = psutil.Process(pid)
+                proc_stats.append((get_rss(proc.memory_info()), pid,
+                                    proc.cmdline()))
+            except (psutil.NoSuchProcess, psutil.AccessDenied, psutil.ZombieProcess) as psex:
+                logger.exception(f"Error obtaining memory info for process {pid}:", psex)
```

Comment:
here we are not adding all processes to the list proc_stats. I recommend adding all processes to the list by, for example, just adding a tuple with an RSS of 0 and CMLLINE with an error message like the one added to the logs (making sure is less than 100 characters giving the limitation below).

Reply:
Done.